services/jupyter-amx.service

```
@@ -0,0 +1,14 @@
+[Unit]
+Description=AMX Jupyter service
+
+[Service]
+User=softioc
+Group=softioc
```

This is done that way at FMX.
```suggestion
User=xf17id1
Group=xf17id1
```